libs/blocks/global-navigation/global-navigation.css

```diff
@@ -373,12 +373,7 @@ header.global-navigation {
 .feds-utilities {
   display: flex;
   align-items: center;
-  padding-right: 8px;
-}
-
-[dir = "rtl"] .feds-utilities {
-  padding-right: 0;
-  padding-left: 8px;
+  padding: var(--feds-gutter);
```

Comment:
Should we do `0 var(--feds-gutter)`? Is there a benefit to having up/down padding as well?

Reply:
you're correct, we need `0 var(--feds-gutter)`